jira-dependency-graph.py

```diff
@@ -83,6 +83,9 @@ def get_status_color(status_field):
     def create_node_text(issue_key, fields, islink=True):
         summary = fields['summary']
         status = fields['status']
+        assignee = ''
+        if show_assignee and 'assignee' in fields and fields['assignee']:
+            assignee = '<BR/><I>' + fields['assignee']['displayName'] + '</I>'
```

Comment:
Since these are not html nodes, this needs to use '\n' as new line character. Also the <I> isn't evaluated, not sure if this is possible to do without html nodes. You could of course change the whole thing to use html nodes instead.